src/components/Settings/SettingsAbout/Releases/index.tsx

```diff
@@ -100,7 +100,7 @@ const Release: React.FC<ReleaseProps> = ({
         </Modal>
       </Transition>
       <div className="flex items-center justify-center mb-4 sm:mb-0 sm:justify-start">
-        <span className="mr-2 text-sm">
+        <span className="mr-2 text-xs mt-1">
```

Comment:
The changes in both this AND the Sonarr modal break the design of the modals. The text boxes are no longer full width.